src/scripts/deploy-script.ts

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env node
+import {
+  AccountRegistry,
+  Chain,
+  ContractRegistryLoader,
+  deployToChain,
+  parseObjFromFile,
+} from "..";
+
+import { getMainLogger } from "../utils/cli";
+import { DEFAULT_RPC_URL } from "../utils/constants";
+
+// can either parse
+const accountConfigFromYaml = {
+  name: "local",
+  registry: parseObjFromFile("specs/evm.accounts.yaml"),
```

Comment:
this works while you are executing the script from the right directory, which I guess is the expected situation when running `npm run deploy`. However, this makes me wonder how will this work when used from `ibctl` for example? Let's keep that in mind anyway, it would be nice not having to duplicate the yaml file :)

Reply:
ah good point - i think we can add env variables to configure - I added them here: https://github.com/open-ibc/vibc-core-smart-contracts/blob/2ff5078d4b3cd165d0ecc4cfe3f22e12319abb2b/src/utils/constants.ts#L27-L37

so for ibctl (or other use cases, we should just be able to specify the `SPECS_BASE_PATH` env variable and that will override the default files

Reply:
hmm can we bundle the contract yaml with the package instead? I would expect not having to set up any of this when running `ibctl`. You should be able to run this 

```sh
$ ibctl deploy ... # uses the bundled "default" yaml

$ ibctl deploy --contracts '/path/to/my-spec.yaml' # in case I want to override the default one. 
```

Reply:
ah yes adding cli options might be a better idea than using env variables. 

The first command should already work but I can work on adding the path as a cli arg instead of an env variable

Reply:
added CLI parsing too now! So the order in which we search for arguments, in order of priority: 

- first search for any args passed into CLI
- then search for ENV variables
- then use default values